src/evidently/tests/data_drift_tests.py

```diff
@@ -502,3 +505,51 @@ def render_html(self, obj: TestColumnDrift) -> TestHtmlInfo:
             )
             info.with_details(f"{column_name}", plotly_figure(title="", figure=fig))
         return info
+
+
+class TestEmbeddingsDrift(Test):
+    name = "Drift for embeddings"
+    group = DATA_DRIFT_GROUP.id
+    metric: EmbeddingsDriftMetric
+    embeddings: str
```

Comment:
it seems that you dont actually set those fields in constructor. either set them or remove those annotations